sockeye/rnn_attention.py

```diff
@@ -707,7 +795,7 @@ def get_context_and_attention_probs(values: mx.sym.Symbol,
                                  axis=1,
                                  use_sequence_length=True,
                                  sequence_length=length,
-                                 value=C.LARGE_NEGATIVE_VALUE)
+                                 value=np.finfo(dtype).min)
```

Comment:
also technically a reason to bump the major version. Practically it probably does not make a difference. However, if we don't bump this should be confirmed.

Reply:
I can revert to the original constant to avoid version bump

Reply:
as above, I think a version bump is fine.